app/lib/hyku_addons/per_tenant_smtp_interceptor.rb

```diff
@@ -10,11 +10,11 @@ def self.delivering_email(message)
       Account.find_by(tenant: Apartment::Tenant.current)&.switch!
       return unless (mailer_settings = Settings.smtp_settings.presence)
 
-      if mailer_settings.from.present?
-        message.from = mailer_settings.from
-        message.reply_to = mailer_settings.from
-        message.return_path = mailer_settings.from
-        message.smtp_envelope_from = mailer_settings.from
+      if (from = mailer_settings.from.presence)
```

Comment:
There are a few other places where a similar pattern is employed, but its a little different and IMO a little clearer to read. 

`if (from = mailer_settings.from).present?`

Can we use that style instead so we keep stuff consistent.